src/Tizen.NUI.Components/Controls/Progress.cs

```diff
@@ -262,6 +265,38 @@ public string BufferImageURL
             }
         }
 
+        /// <summary>
+        /// The property to get/set the indeterminate image.
+        /// </summary>
+        /// <exception cref="NullReferenceException">Thrown when setting null value.</exception>
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string IndeterminateImage
+        {
+            get
+            {
+                if (indeterminateImage == null)
+                {
+                    return null;
+                }
+                else
+                {
+                    return ((string)indeterminateImage.URL);
```

Comment:
I think this can be shortened like this =>
`return indeterminateImage?.URL;`
( https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and- )
how do you think?

Reply:
Yes, you're right. I'll update it as you suggested.